src/App.js

```diff
@@ -1,23 +1,48 @@
+import {useState, useRef} from 'react';
 import styled from 'styled-components';
 
 import MagicForm from './components/MagicForm/MagicForm';
+import MagicList from './components/MagicList/MagicList.js';
+import {CodeSpan} from './components/Stylesheet/StyledSpans.js';
 
 export default function App() {
-  //
-  function analyzeSpell(spellword) {
+  // State // for the archived Spells
+  const [savedOrders, setSavedOrders] = useState([]);
+
+  //useRef is used here to find the MagicList at the End of following saveSpellOrder-function
+  const ref = useRef();
+
+  // ANALYZE SPELL // checks the Input.value from MagicForm and sends back and to: MagicList //
+  function saveSpellOrder(spellword) {
+    //
+    const trimmedInput = spellword.trim();
+    const checkedInput = trimmedInput.toLowerCase();
+    const inputInfo = findSpellMessage(checkedInput);
+    const hasError = inputInfo.includes('ERROR'); //the error-prop is used to make specific text red, the string can tell if error has happened
+    setSavedOrders([{value: spellword, info: inputInfo, error: hasError}, ...savedOrders]);
+
+    // The following parts set the scrollingPosition inside the List to bottom on new entry:
+    const List = ref.current;
+    const topPos = List.offsetTop;
+    List.scrollTop = topPos;
+  }
+
+  // ↑ FIND SPELL MESSAGE // is called within analyzeSpell to return a fitting string //
+  function findSpellMessage(spellword) {
     if (spellword === 'cd') {
       return 'The command "cd" stands for "change directory", so you can jump from one directory in another one. This is so important. ';
     } else if (spellword === 'ls') {
       return ' The command "ls" stands for "list items", it shows you all directories and data, that are stored in the directory you are in right now and are not hidden.';
     } else if (spellword === 'pwd') {
       return ' The command "pwd" stands for "print working directory", this means that you can ask for your current position and it will tell you the directory you are in right now. ';
     } else {
-      return '(!)>>> ERROR: please check if you spelled the command correctly, coding is really strict and otherwise it will not work.';
+      return '(!) ERROR: please check if you spelled the command correctly, coding is really strict and otherwise it will not work.';
     }
   }
 
   return (
     <OrganizingMain>
+      <LogoImg src={require('./images/logo1.png')} alt="" width="125" height="125" />
```

Comment:
You can set the value "Logo" for the alt-tag

Reply:
How @Quaquaro said, the alt attribute should not be empty.

Reply:
If your app name is only shown inside the logo and nowhere else, it would be good to write the app name in the alt-attribute :)